### PR TITLE
feat(db): more verbosity on importing

### PR DESCRIPTION
**Pull request by Mte90**
Adds more verbosity during importing, I am not sure if we can add more.

## Checks

* [ ] I've updated the changelog.
* [x] I've tested this PR
* [x] This PR is for the `develop` branch not the `stable` branch.
* [ ] This PR is complete and ready for review.

### CODE REVIEW

**Comment by Mte90** [comment]
Basically improve the `unbound variable` and set in case the variable is not set, so during provision is able to add the log like should be.

**Comment by Mte90** [comment]
Tested and work on provision and also if launched manually, covers both the cases of the patch of few days ago.

**Comment by Mte90** [comment]
@tomjn can you approve that one?

**Review by tomjn — DISMISSED** [review]

**Comment by tomjn** [comment]
@Mte90 needs changelog update

**Comment by Mte90** [comment]
Changelog updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ permalink: /docs/en-US/changelog/
 * Improved provisioning output
 * VVV will now attempt to test Nginx configs on installation and recover ( #2604 )
 * Switched to new launchpad PPA domains with HTTPS ( #2586 )
+* Improved verbosing in DB import ( #2621 )
```
Comment (tomjn) [inline CHANGELOG.md:16]:
```suggestion
* Improved the verboseness of the DB import scripts ( #2621 )
```

**Review by tomjn — APPROVED** [review]

**Pull request merged**